### PR TITLE
Fix repo setup tests for missing maintain beta and not required ansible repo for el8

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
**Tests Results:**
```
(.testfm) ➜  testfm git:(remove-ansible-el8) ✗ pytest -q --disable-pytest-warnings --ansible-host-pattern server --ansible-user root --ansible-inventory testfm/inventory tests/test_advanced.py -k test_positive_satellite_repositories_setup
.                                                                                                                                                                     [100%]
1 passed, 13 deselected in 189.68 seconds
```
```
(.testfm) ➜  testfm git:(remove-ansible-el8) ✗ pytest -q -m capsule --disable-pytest-warnings --ansible-host-pattern server --ansible-user root --ansible-inventory testfm/inventory tests/test_advanced.py -k test_positive_capsule_repositories_setup
.                                                                                                                                                                     [100%]
1 passed, 13 deselected in 190.06 seconds
```

### CODE REVIEW

File: tests/test_advanced.py [hunk 1]
```diff
@@ -394,30 +395,34 @@ def test_positive_satellite_repositories_setup(setup_subscribe_to_cdn_dogfood, a
             for repo in sat_repos[ver]:
                 assert repo in result["stdout"]
 
+    # 6.12 till not GA
+    contacted = ansible_module.shell(Advanced.run_repositories_setup({"version": "6.12"}))
```
Comment (vsedmik) [inline tests/test_advanced.py:399]: If there is no special need to use both, I would suggest to use just one of the `command`/`shell` ansible module.

**Review by vsedmik — APPROVED** [review]
Looks good to me, left just one non-blocking comment.

File: tests/test_advanced.py (hunk 1 above)
Comment (jameerpathan111) [reply tests/test_advanced.py:399]: Preferably `command` but I know sometimes we need to use `shell` instead.

**Review by jameerpathan111 — APPROVED** [review]

File: tests/test_advanced.py (hunk 1 above)
Comment (shubhamsg199) [reply tests/test_advanced.py:399]: yes, Using command module, we cannot use variables like $HOSTNAME and operations like  ">",  "|",  ";" ,  etc with shell we can.

**Review by shubhamsg199 — APPROVED** [review]

File: tests/test_advanced.py (hunk 1 above)
Comment (Gauravtalreja1) [reply tests/test_advanced.py]: @vsedmik Good catch, update this step to use the command module, and for beta repos part we need shell module for export command and ";"

**Pull request merged**